src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -358,14 +450,30 @@ void DefaultDACVerifier::VerifyAttestationInformation(const DeviceAttestationVer
         }
     }
 
+    // Log info about the DAC chain so far (DAC, PAI).
+    if (AreVerboseLogsEnabled())
+    {
+        ChipLogProgress(NotSpecified, "Device candidate DAC chain details:");
+        ChipLogProgress(NotSpecified, "--> DAC's VID: 0x%04X, PID: 0x%04X", dacVidPid.mVendorId.Value(),
+                        dacVidPid.mProductId.Value());
+
+        LogCertificateAsPem(AttestationChainElement::kDAC, info.dacDerBuffer);
+        LogOneKeyId(KeyIdType::kSubjectKeyId, AttestationChainElement::kDAC, info.dacDerBuffer);
+        LogOneKeyId(KeyIdType::kAuthorityKeyId, AttestationChainElement::kDAC, info.dacDerBuffer);
```

Comment:
Could factor this out into a `LogAttestationChainElement(AttestationChainElement, ByteSpan)` thing, so we don't have three copies of those three lines, but either way.

Reply:
Done